src/Test/Utilities/Portable/CommonTestBase.cs

```diff
@@ -194,19 +194,24 @@ internal CompilationVerifier CompileAndVerifyFieldMarshal(string source, Func<st
 
         static internal void RunValidators(CompilationVerifier verifier, Action<PEAssembly> assemblyValidator, Action<IModuleSymbol> symbolValidator)
         {
+            var emittedMetadata = verifier.GetMetadata();
```

Comment:
>var emittedMetadata = verifier.GetMetadata(); [](start = 12, length = 45)

We probably shouldn't even do this if there are no validators to run.

Reply:
I see that there is only one call site and it was adjusted to not call this method if there no validators. Consider adding an assert here.

---
In reply to: [155404430](https://github.com/dotnet/roslyn/pull/22896#discussion_r155404430) [](ancestors = 155404430)